src/main/resources/assets/js/components/DeploymentComponent.jsx

```diff
@@ -23,8 +23,12 @@ define([
       this.setState({loading: bool});
     },
 
-    handleDestroyDeployment: function () {
-      this.props.destroyDeployment(this.props.model, this);
+    handleDestroyDeployment: function(forceStop) {
+      return function() {
+        var model = this.props.model;
+        model.setForceStop(forceStop);
+        this.props.destroyDeployment(model, this);
```

Comment:
Pass in the forceStop here in stead: `this.props.destroyDeployment(model, {forceStop: forceStop}, this);`